Jellyfin.Server/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "Jellyfin.Server": {
+      "commandName": "Project"
+    },
+    "Jellyfin.Server (nowebcontent)": {
+      "commandName": "Project",
+      "commandLineArgs": "--nowebcontent"
+    }
```

Comment:
This adds a separate debugging profile to easily run with the command line flag set when running from an IDE. Tested in Visual Studio only, but I'm fairly sure VSCode integrates with this file as well.

![image](https://user-images.githubusercontent.com/2626103/76707105-d3cd8d00-66ec-11ea-9738-cbd443576a16.png)